src/aiperf/workers/worker.py

```diff
@@ -194,8 +211,18 @@ async def _execute_single_credit_internal(
 
         turn_list = []
         for turn_index in range(len(conversation.turns)):
-            self.task_stats.total += 1
+            # Apply turn delay BEFORE sending the turn (simulating user thinking time)
+            # Skip delay for the first turn
```

Comment:
is it a hard and fast rule to skip the turn delay on the first turn? I feel like that would be something more to guard against in the data loader. but also, it doesn't make sense to have a delay for the first turn. so idk. I think it is already guarded against anyways.